src/App.css

```diff
@@ -0,0 +1,121 @@
+@import url("https://fonts.cdnfonts.com/css/gill-sans");
+@import url("https://fonts.googleapis.com/css?family=Lato:400");
+
+body {
+  background-color: rgb(147, 112, 219);
+  font-family: "Gill Sans", "lato", sans-serif;
+}
+
+.link {
+  text-decoration: none;
+}
+
+.mainContainer {
+  background-color: rgb(147, 112, 219);
+  margin-top: 50px;
+}
+
+.year {
+  color: white;
+  font-size: medium;
+  margin-left: 3px;
+}
+
+.searchBox {
+  margin-right: 0;
+  margin-left: 20%;
+}
+
+.searchButton {
+  margin-right: 55px;
+  padding-right: 22px;
+  width: 25px;
+}
+
+.title {
+  margin-left: 20%;
+  margin-bottom: 50px;
+  color: white;
+}
+
+.homeCoinList {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  list-style: none;
+  margin-top: 20px;
+  padding-right: 8rem;
+}
```

Comment:
- [ ] Good job taking the optional suggestion of the previous reviewer to add padding to this section, however, there is now an overflow as shown below. Kindly fix it
<img width="372" alt="Screenshot 2023-03-24 at 18 39 40" src="https://user-images.githubusercontent.com/64774936/227600342-0ed7cbee-b7e3-468b-b130-91c1bc142351.png">